SU-HSDO/suhumsci
# SHS 5154 layout builder theming

# READY FOR REVIEW

## Summary
This PR: 
- Ensures checkboxes are close to their corresponding field labels in layout builder
- Ensures the Revision Information section is closed by default.
- Ensures that the field label and field is clear and distinguishable from other fields. 

## Need Review By (Date)
_['12/12']_

## Urgency
_['medium']_

## Steps to Test
1. Visit a page that uses layout builder (like the homepage)
2. Edit the homepage and click layout
3. Verify that checkboxes have been themed to be smaller and next to their label
4. Verify that the revision information detail element is closed by default
5. Note that the fields and labels are better distinguished (grouped together, with more margin between each pair.) 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
